src/assignments/html-me-somthing.rst

```diff
@@ -0,0 +1,338 @@
+Assignment #4: HTML Me Someting
+================================
+
+You’ve learned a bit of HTML and some CSS, but you have likely only used
+it in bits and pieces so far, adding or modifying content in exercises
+or pre-existing files. Here, you are going to take another step forward
+by building an entire page from scratch. You will also get some practice
+using Git.
+
+There are two parts to this exercise, one focused on HTML and another
+focused on CSS. HTML makes up the *structure and content* of web pages,
+while CSS dictates the *visual style*.
+
+Best practices dictate that *content* and *style* should be kept as separate as
+possible. To that end, we will build the HTML portion of our page first,
+and afterwards we will add a few styles with CSS. This will help us avoid
```

Comment:
I am confused by the final sentence: "This will help us avoid creating HTML elements simply to change the style of our page."

State what "this" is to enhance clarity.

Reply:
It could be fine after re-reading it. But it's worth further consideration.

Reply:
Good point.  I added a longer rationale to try to increase the clarity.  Let me know what you think.